docs/planning/QuestionPlan.md

```diff
@@ -0,0 +1,55 @@
+    *= preventing unauthorized disclosure of information. Includes secrecy, privacy*
+    *= preventing downtime of systems or inability to access data/information*
+    *= preventing unauthorized modification of data and systems*
+
+- "Looking at this clue here reminds me of one of the CIA triad elements, I remember a definition, 'it prevents unauthorised modification of data and systems' but which one of the elements does it belong, let me think."
+    *= Integrity*
+    *= Availability*
+    *= Identity*
+
+- "This clue is fascinating I think it's to do with the availability element of the CIA triad. I should check by looking at the definition. Select the definition of availability"
+    *= preventing downtime of systems or inability to access data/information*
+    *= preventing unauthorized modification of data and systems*
+    *= preventing unauthorized disclosure of information. Includes secrecy, privacy*
+
+- "Hmmm, it looks like the victim, Ginny, had granted someone permission to access a file, what is the name of this process?"
+    *= Authorisation*
+    *= Authentication*
+    *= Apple bobbing*
+
+- "I wonder if the hacker managed to gain authentication? Select the definition of authentication"
+    *= the process of confirming the truth or correctness of the claimed artefact or identity*
+    *= the process of granting permission to someone/thing do some action (e.g., access files)*
+    *= the process of hanging a picture on the wall*
+
+- "This clue looks linked to the factor Non-repudiation to me. Select the definition of non-repudiation"
+    *= ability for parties to prove that a message has been sent by a specific person, and received by a specific person. Therefore neither party can claim they did not send/receive the message.*
+    *= the process of granting permission to someone/thing do some action (e.g., access files)*
+    *= the process of confirming the truth or correctness of the claimed artefact or identity*
+
+- "This clue shows we should think about the computer security of this mansion. Select which of the following describes what computer security is"
+    *= A measure of how well a system resists misuse from either an insider or an outsider*
+    *= An apple a day keeps the doctor away*
+    *= A measure of how well a system stores files*
+ 
+- "Hmmm let's look at cyber security in more detail. The collection of what can be used to protect the cyber environment, organisation and user's assets?"
+    *= Collection of tools, policies, security concepts, security safeguards, guidelines, risk management approaches, actions, training, best practices, assurance and technologies*
+    *= Collection of connected computing devices, personnel, infrastructure, applications, services, telecommunications systems*
+    *= Collection of scout badges*
+
+- "This clue seems to point in the direction of computer security. Which of the following describes what computer security is?"
```

Comment:
This question seems to be the same as the one on L42, but with different answers. That could be a little confusing